apps/web/playwright/organization/organization-invitation.e2e.ts

```diff
@@ -9,12 +9,12 @@ import { expectInvitationEmailToBeReceived } from "./expects";
 
 test.describe.configure({ mode: "parallel" });
 
-test.afterEach(async ({ users, emails }) => {
+test.afterEach(async ({ users, orgs }) => {
   await users.deleteAll();
-  emails?.deleteAll();
```

Comment:
No need to delete emails separately. users.deleteAll would take care of emails cleanup